src/service.js

```diff
@@ -1,4 +1,4 @@
-const fetchEnterpriseCustomers = apiClient => apiClient.get(`${process.env.LMS_BASE_URL}/enterprise/api/v1/enterprise-customer/`);
```

Comment:
can this defaulting be only done in the function above or is also needed here? ideally we could read this in one place defaulted to that env var.. but this is not a big issue, just that we have two places to change in case we use a different env var or sth in the future

Reply:
So the reason I left this here is because this function is importable. There's a scenario where you could `import { fetchEnterpriseCustomers } from 'frontend-enterprise/service'` and would then have the option of passing along a var or not depending on if it'll be set.

Reply:
I see, that makes sense... I can see a future mode where we have a function whose sole job is to resolve these values and both functions use it, and then we test that function. But it would not be a value investment for this stage of work I think, so all set